rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/uri/jsonrpc/messages/ISourceLocationChangeType.java

```diff
@@ -23,15 +25,15 @@ public static ISourceLocationChangeType forValue(int value) {
         return allValues[value - 1];
     }
 
-    public static org.rascalmpl.uri.ISourceLocationWatcher.ISourceLocationChangeType translate(
```

Comment:
I have the feeling that I wrote this code before. Probably a translation in the other direction ;-) But let's check if we didn't duplicate a whole hierarchy.

Reply:
This is by design a duplicate of the enum inside the ISourceLocationWatcher. Specifically, for enums that have to be serialized, we want to have them have a specific (predictable) numeric value, that translates to the enums on the vscode side.

Alternativly we could change the definition of the enums inside the ISourceLocationWatcher, but this way would allow for parallel evolution.

Reply:
ah, src\main\java\org\rascalmpl\vscode\lsp\IRascalFileSystemServices.java contains the same enum. okay, I'll try to merge it.

Reply:
I see; it's layering and cloning. too bad but this is Java land so let's keep the current design. please add a comment that refers with @see to the API that we are mirroring here.

Reply:
ok. I'd rather move ahead now; let's add an issue in the current project to remind us of this TD.

Reply:
Made an issue #160 .